sycl/test-e2e/Matrix/element_wise_ops.cpp

```diff
@@ -7,6 +7,8 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: aspect-ext_intel_matrix
 
+// XFAIL: gpu-intel-dg2
+
```

Comment:
Same as above:
Adding XFAIL will make the code fail silently for the other supported combination.